kobolds/biology.md

```diff
@@ -1,58 +1,66 @@
-## Physical Appearance
+# Kobold Biology
 
-### Anatomy
+1. [Anatomy](#anatomy)
+   1. [Basic Appearance](#basic-appearance)
+   1. [Scales & Coloration](#scales--coloration)
+1. [Physiology](#physiology)
+   1. [Reproduction](#reproduction)
+   1. [Gestation & Development](#gestation--development)
+
+
+## Anatomy
```

Comment:
Add a section on disease. It's very rare, but mention scale rot, which is persistent and awful.

Reply:
Oof. Scale rot ended up basically being leprocy.